stackit/internal/services/loadbalancer/loadbalancer/datasource.go

```diff
@@ -0,0 +1,318 @@
+		"protocol":             "Protocol is the highest network protocol we understand to load balance.",
+		"target_pool":          "Reference target pool by target pool name.",
+		"name":                 "Load balancer name.",
+		"networks":             "List of networks that listeners and targets reside in.",
+		"network_id":           "Openstack network ID.",
+		"role":                 "The role defines how the load balancer is using the network.",
+		"options":              "Defines any optional functionality you want to have enabled on your load balancer.",
+		"acl":                  "Load Balancer is accessible only from an IP address in this range.",
+		"private_network_only": "If true, Load Balancer is accessible only via a private network IP address.",
+		"private_address":      "Transient private Load Balancer IP address. It can change any time.",
+		"target_pools":         "List of all target pools which will be used in the Load Balancer. Limited to 20.",
+		"healthy_threshold":    "Healthy threshold of the health checking.",
+		"interval":             "Interval duration of health checking in seconds.",
+		"interval_jitter":      "Interval duration threshold of the health checking in seconds.",
+		"timeout":              "Active health checking timeout duration in seconds.",
+		"unhealthy_threshold":  "Unhealthy threshold of the health checking.",
+		"target_pools.name":    "Target pool name.",
+		"target_port":          "Identical port number where each target listens for traffic.",
+		"targets":              "List of all targets which will be used in the pool. Limited to 250.",
+		"targets.display_name": "Target display name",
+		"ip":                   "Target IP",
+	}
+
+	resp.Schema = schema.Schema{
+		Description: descriptions["main"],
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Description: descriptions["id"],
+				Computed:    true,
+			},
+			"project_id": schema.StringAttribute{
+				Description: descriptions["project_id"],
+				Required:    true,
+				Validators: []validator.String{
+					validate.UUID(),
+				},
+			},
+			"external_address": schema.StringAttribute{
+				Description: descriptions["external_address"],
+				Computed:    true,
+			},
+			"listeners": schema.ListNestedAttribute{
+				Description: descriptions["listeners"],
+				Computed:    true,
+				Validators: []validator.List{
+					listvalidator.SizeBetween(1, 20),
+				},
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"display_name": schema.StringAttribute{
+							Description: descriptions["listeners.display_name"],
+							Computed:    true,
+						},
+						"port": schema.Int64Attribute{
+							Description: descriptions["port"],
+							Computed:    true,
+						},
+						"protocol": schema.StringAttribute{
+							Description: descriptions["protocol"],
+							Computed:    true,
+							Validators: []validator.String{
```

Comment:
In the datasource, I dont see why we should have these oneOf validators. If it is returned by the API, it should be a valid value. There is no need to have to maintain this list here

Reply:
Agree, copy paste leftover